tree/dataframe/inc/ROOT/RDF/RInterface.hxx

```diff
@@ -3025,7 +3025,7 @@ public:
    /// \param[in] nMaxCollectionElements Maximum number of collection elements to display per row.
    /// \return the `RDisplay` instance wrapped in a RResultPtr.
    ///
-   /// This function returns a RResultPtr<RDisplay>` containing all the entries to be displayed, organized in a tabular
+   /// This function returns a RResultPtr<RDisplay> containing all the entries to be displayed, organized in a tabular
```

Comment:
```suggestion
   /// This function returns a RResultPtr< RDisplay > containing all the entries to be displayed, organized in a tabular
```
To avoid doxygen warning.
Alternatively, use:
```
`RResultPtr<RDisplay>`
```